express-notes-api/index.js

```diff
@@ -0,0 +1,130 @@
+const express = require('express');
+const app = express();
+const fs = require('fs');
+const jsonNotes = require('./data.json');
+
+app.get('/api/notes', (req, res) => {
+  const notesArray = [];
+  for (const eachNote in jsonNotes) {
+    const outer = jsonNotes[eachNote];
+    for (const key in outer) {
+      notesArray.push(outer[key]);
+    }
+  }
```

Comment:
Since you are interested in a particular part of your data model, just loop through that.

```js
for (const id in jsonNotes.notes) {
```